.cursor/rules/build-run-test.mdc

```diff
@@ -0,0 +1,18 @@
+---
+description:
+globs:
+alwaysApply: true
+---
+# Build, Run, Test
+
+You may need to learn how to build and test RisingWave when implementing features or fixing bugs.
+
+- Use `./risedev b` to build the project.
+- Use `./risedev c` to check if the code follow rust-clippy rules, coding styles, etc.
+- Use `./risedev d` to run a RisingWave instance. If the project is not built, this will build the project before running. The instance will run in backgroud, so you don't need to start a new terminal to run this command. You can connect to the instance right after this command is finished.
+- Use `./risedev k` to stop a RisingWave instance started by `./risedev d`.
```

Comment:
I'm a little hesitant to put `risedev d/slt` in the same file as `risedev b/c`. 

As getting compilation passed is relatively straightforward and can be automated. But running e2e tests is more complex. Might need to teach: how to write tests, how to check if the cluster is running, etc

Reply:
Can merge and refine if it doesn't work welll

Reply:
Ya feel free to update the rules